src/vcf/record.cpp

```diff
@@ -236,6 +236,16 @@ namespace ebi
         if (std::find(format.begin(), format.end(), "GT") != format.end() && format[0] != "GT") {
             throw new FormatBodyError{line, "GT must be the first field in the FORMAT column"};
         }
+
+        if (format.size() > 1) {
```

Comment:
when format == 0 the function has already returned before this code, unless size can be negative it is always gonna be >1 right?